easybuild/easyconfigs/n/NCCL/NCCL-2.1.4-CUDA-9.0.176.eb

```diff
@@ -0,0 +1,29 @@
+easyblock = "MakeCp"
+
+name = 'NCCL'
+version = '2.1.4'
+cuda_version = '9.0.176'
+versionsuffix = '-CUDA-%s' % cuda_version
+
+homepage = 'https://developer.nvidia.com/nccl'
+description = """The NVIDIA Collective Communications Library (NCCL) implements multi-GPU and multi-node collective
+communication primitives that are performance optimized for NVIDIA GPUs."""
+
+toolchain = {'name': 'dummy', 'version': ''}
+
+sources = ['%%(namelower)s_%%(version)s-1+cuda%s_x86_64.txz' % '.'.join(cuda_version.split('.')[0:2])]
```

Comment:
@damianam Please include a comment to clarify where to download from (https://developer.nvidia.com/nccl/nccl-download).

Also: include SHA256 checksum